src/defaults/fileIcons.ts

```diff
@@ -961,7 +961,10 @@ const fileIcons: FileIcons = {
     ],
   },
   'gitlab': {
-    fileExtensions: ['gitlab-ci.yml'],
+    fileExtensions: [
+      'gitlab-ci.yml',
+      'gitlab-ci.yaml',
+    ],
```

Comment:
I'm a bit confused why this is in the `fileExtensions` block in the first place. Is `gitlab-ci.yml` itself a file extension?

Reply:
It can be, some people do something like `rust.gitlab-ci.y(a)ml`, but more commonly it's used either in the riot directory as `.gitlab-ci.y(a)ml` or in `.gitlab/.gitlab-ci.y(a)ml`